spec/services/scm/create_managed_repository_service_spec.rb

```diff
@@ -134,4 +136,43 @@
       end
     end
   end
+
+  context 'with managed remote config' do
+    include_context 'with webmock'
```

Comment:
Thinking about it, i would expect to receive some state from the context (e.g., a new `let` as in the tmpdir shared context).

I'll refactor it to metadata.